CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ set(
   "BN128"
   CACHE
   STRING
-  "Default curve: one of ALT_BN128, BN128, EDWARDS, MNT4, MNT6"
+  "Default curve: one of ALT_BN128, BN128, EDWARDS, MNT4, MNT6, BLS12_381"
```

Comment:
Could you add BLS12_377 and BW6_761 too please?  This parameter should really be removed in libff, but while it exists we should maintain it.  Thanks.